selfdrive/car/chrysler/values.py

```diff
@@ -166,14 +166,14 @@ class ChryslerCarInfo(CarInfo):
       bus=0,
     ),
   ],
+  ecus={(0x742, None): Ecu.combinationMeter, (0x744, None): Ecu.srs, (0x747, None): Ecu.abs,
+        (0x753, None): Ecu.fwdRadar, (0x75a, None): Ecu.eps, (0x7e0, None): Ecu.engine,
+        (0x7e1, None): Ecu.transmission, (0x761, None): Ecu.eps},
+  # Ecus present in hybrids, currently in data collection mode
+  extra_ecus={(0x7e2, None): Ecu.hcp, (0x7e4, None): Ecu.abs},
```

Comment:
If we don't really care about being too careful, we can just add these ecus on to the main ecu dictionary. I don't feel too strongly one way or the other (together with a comment separating them, or another field like `extra_ecus` and have a tests for all this)

@adeebshihadeh thoughts?